languages.toml

```diff
@@ -1781,3 +1781,14 @@ language-server = { command = "bass", args = ["--lsp"] }
 [[grammar]]
 name = "bass"
 source = { git = "https://github.com/vito/tree-sitter-bass", rev = "501133e260d768ed4e1fd7374912ed5c86d6fd90" }
+
+[[language]]
+name = "wat"
```

Comment:
I think it would make more sense to call this `webassembly`, but it seems that this name determines the symbol that will be imported from the grammer dynamic library, and that's fixed to `tree_sitter_wat`.

Reply:
There's an issue along those lines: https://github.com/helix-editor/helix/issues/3164

It would only be a cosmetic improvement for the file-type statusline element though